app/forms/common/YesNoAnswer.scala

```diff
@@ -34,7 +34,10 @@ object YesNoAnswer {
 
   import YesNoAnswers._
 
-  val allowedValues: Seq[String] = Seq(yes, no)
+  lazy val allowedValues: Seq[String] = Seq(yes, no)
+
+  lazy val Yes = YesNoAnswer(yes)
+  lazy val No = YesNoAnswer(no)
```

Comment:
What is the reason for making them lazy? These fields use almost no computational and space resources.